.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Proposal Follow-up
+about: Follow-up issue for a proposal
+title: ''
+labels: 'active proposal'
+assignees: ''
+
+---
+
+**Which proposal does this relate to?**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Is this a copy/paste error? Looks like the same text below. Shouldn't it say something like 

> Name the proposal this follow-up refers to using the XXXX-ProposalName format.